flow/layers/layers.proto

```diff
@@ -0,0 +1,124 @@
+syntax = "proto3";
```

Comment:
can you split this file into multiple files, one per protocol, DNS, DHCP, VRRP

Reply:
done